plugin/trino-cassandra/pom.xml

```diff
@@ -68,6 +66,62 @@
             <artifactId>units</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.datastax.oss</groupId>
+            <artifactId>java-driver-core</artifactId>
+            <version>${dep.casandra.version}</version>
+            <exclusions>
```

Comment:
I think we should update https://github.com/trinodb/trino-cassandra-driver/blob/master/pom.xml instead of doing exclusions. Exclusion can easily cause classloader issues.

Reply:
I came to exclude `asm` because trino uses the version `6.1` and cassandra java driver uses the version  `9.1`.
I do agree with you that doing this may cause an unwanted issue at runtime.

Please also note that the issue https://github.com/trinodb/trino/issues/7729 that this PR is addressing is about ditching `trino-cassandra-driver` and replacing it with the vanilla cassandra java driver.

Pls also see the related Slack discussion https://trinodb.slack.com/archives/CP1MUNEUX/p1619119955409200

Reply:
We should upgrade Trino to latest ASM.

Reply:
@electrum you have recently worked on https://github.com/airlift/bytecode for upgrading it to the latest ASM.
Are there any blockers for bringing the changes in Trino? (migrate `io.airlift:bytecode` from `1.2` to  `1.4` in `trino/pom.xml` ?

Reply:
No, should be fine update.

Reply:
@ebyhr  i have bumped up bytecode to `1.4` in trino's pom.xml and rebased therefore this PR. Could you PTAL?